frontend/src/components/Contexts/SafeZoneContext.tsx

```diff
@@ -10,20 +16,50 @@ interface Props {
 }
 
 const defaultSafeZoneInterface = {
-    safeZoneStatus: JSON.parse(localStorage.getItem('safeZoneStatus') ?? 'false'),
+    safeZoneStatus: false,
     switchSafeZoneStatus: (newSafeZoneStatus: boolean) => {},
 }
 
 export const SafeZoneContext = createContext<ISafeZoneContext>(defaultSafeZoneInterface)
 
 export const SafeZoneProvider: FC<Props> = ({ children }) => {
     const [safeZoneStatus, setSafeZoneStatus] = useState<boolean>(defaultSafeZoneInterface.safeZoneStatus)
+    const { enabledRobots } = useRobotContext()
+    const { installationCode } = useInstallationContext()
+    const { setAlert } = useAlertContext()
 
     const switchSafeZoneStatus = (newSafeZoneStatus: boolean) => {
```

Comment:
This is not important since it's from an earlier PR, but would it maybe be better to just use setSafeZoneStatus? Since we are giving the value we want for it anyways? Alternatively we could keep it as switchSafeZoneStatus but have it instead set the status to setSafeZoneStatus((oldStatus) => !oldStatus). Currently switchSafeZoneStatus is identical to setSafeZoneStatus